<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="bugTab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1001</width>
      <height>581</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Bug Tracker</string>
     </attribute>
     <widget class="QTableWidget" name="issueTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>971</width>
        <height>501</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Priority</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Future Developments Tracker</string>
     </attribute>
     <widget class="QTableWidget" name="scheduleTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>991</width>
        <height>501</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Priority</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="projLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Project: None Loaded</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionAddIssue"/>
    <addaction name="actionAddSchedule"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionRemoveTask"/>
    <addaction name="actionUpdatePriority"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProject"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Issue">
   <property name="text">
    <string>Add Bug</string>
   </property>
  </action>
  <action name="actionAdd_Schedule">
   <property name="text">
    <string>Add Schedule</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionAddIssue">
   <property name="text">
    <string>Add Issue</string>
   </property>
  </action>
  <action name="actionAddSchedule">
   <property name="text">
    <string>Add Future Development</string>
   </property>
  </action>
  <action name="actionRemoveTask">
   <property name="text">
    <string>Remove task</string>
   </property>
  </action>
  <action name="actionUpdatePriority">
   <property name="text">
    <string>Update Priority</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
